### PR TITLE
DOCS-2223: Add missing brace in AMEX JSON

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/api/payment_methods/post-order-american-express.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ meta_description: "The MultiSafepay Documentation Center presents all relevant i
   },
   "customer": {
     "locale": "nl_NL",
-    "ip_address": "127.0.0.1"
+    "ip_address": "127.0.0.1" }
```
Comment (Atiq-MultiSafepay) [inline content/api/payment_methods/post-order-american-express.md:26]: @FrancescaNCook I'm not entirely sure if the  } should be after the ip_address. Would be best to check this with Kris.

**Review by Atiq-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/api/payment_methods/post-order-american-express.md (hunk 1 above)
Comment (FrancescaNCook) [reply content/api/payment_methods/post-order-american-express.md:26]: Ok. I ran both the existing JSON and the fix provided in the ticket through a JSON checking tool and it agreed that the brace was missing. Your thoughts @Kris-MultiSafepay ?

Comment (Kris-MultiSafepay) [reply content/api/payment_methods/post-order-american-express.md:26]: Brace was indeed missing, but should be on a new line (same indentation as line 24). Also, lines 12-22 are indented too far (2 spaces).

Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-american-express.md:26]:
```suggestion
    "ip_address": "127.0.0.1"
  }
```

**Pull request merged**